onboarding/serializers.py

```diff
@@ -386,6 +395,3 @@ class UserProgressSerializer(serializers.ModelSerializer):
     class Meta:
         model = Answer
         fields = '__all__'
-        # depth = 3
```

Comment:
I prefer to keep it for _educational_ purpose.
Sometimes I need to uncomment this to see whole relations for debugging also.